test/e2e/shared/exec.go

```diff
@@ -54,13 +54,13 @@ func executeCommands(ctx context.Context, artifactsFolder string, debug bool, lo
 	}
 
 	for _, cmd := range commands {
-		if err := executeCommand(ctx, signer, debug, logDir, bastionIP, machineIP, cmd); err != nil {
+		if err := executeCommand(ctx, signer, debug, logDir, bastionIP, machineIP, machineUser, cmd); err != nil {
 			_, _ = fmt.Fprintln(GinkgoWriter, err.Error())
 		}
 	}
 }
 
-func executeCommand(ctx context.Context, signer ssh.Signer, debug bool, logDir, bastionIP, machineIP string, cmd command) error {
+func executeCommand(ctx context.Context, signer ssh.Signer, debug bool, logDir, bastionIP, machineIP, machineUser string, cmd command) error {
```

Comment:
A struct to pass all this arguments would be nice here. 4 strings arguments right after the other seem like asking for troubles :smile: